api/AnimationTimeline.json

```diff
@@ -51,15 +51,33 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationTimeline/currentTime",
           "support": {
-            "webview_android": {
-              "version_added": "54"
-            },
-            "chrome": {
-              "version_added": "54"
-            },
-            "chrome_android": {
-              "version_added": false
-            },
+            "webview_android": [
+              {
+                "version_added": "61"
```

Comment:
Actually 61 was when the name change occurred, so you've got the versions backwards. The original added should be `true`. Support for `currentTime` predates the name change by 3 years, but I cannot nail down a specific version. It appears to have been some time after we forked from Webkit. 

This note should be at the top of the interface. @Elchi3, would this need to be on both the interface and it's members, or can the rendering script make general notes inherited?

Reply:
something like this?
```js
            "chrome_android": [
              {
                "notes": "Implemented previously as <code>DocumentationTimeline</code>",                
                "version_added": "61"
              },
              {
                "version_added": true
              }
            ],
```

Reply:
Factually that's correct.

Reply:
I haven't documented Web Animations, so not a subject matter expert, but I think `AnimationTimeline` and `DocumentTimeline` are two different things. They both exists and are valid interfaces as far as I can tell from the spec:
- https://drafts.csswg.org/web-animations/#the-animationtimeline-interface
- https://drafts.csswg.org/web-animations/#the-documenttimeline-interface

Firefox also implements both:
- https://dxr.mozilla.org/mozilla-central/source/dom/webidl/AnimationTimeline.webidl
- https://dxr.mozilla.org/mozilla-central/source/dom/webidl/DocumentTimeline.webidl

Are you sure Chrome doesn't have both as well? I see these:
- https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/animation/AnimationTimeline.idl
- https://cs.chromium.org/chromium/src/third_party/WebKit/Source/core/animation/DocumentTimeline.idl

Reply:
Good catch. (I didn't look closely enough at a development commit. [It's here](https://codereview.chromium.org/267353004) if you're curious.

Reply:
Ok where does this leave our PR?

* Remove references to DocumentTimeline in compat JSON
* Return to chrome version 61 in AnimationTimeline.currentTime

Reply:
Yes, remove references to DocumentTimeline.

currentTime has been available since Chrome 39/Opera 26. https://www.chromestatus.com/features/5633748733263872